basm.go

```diff
@@ -57,6 +62,14 @@ type VerifyAttestationOutput struct {
 	RawClaims []byte
 }
 
+const (
```

Comment:
Used in https://github.com/blocky/attestation-service-examples/blob/64f446156ff53f36b5d93bf7c81995c16a1718c5/time_weighted_average_price/attest_fn_call/main.go#L57

Reply:
Fix design.  As implemented, this leaks the internal representation and serialization of types to the user.

Reply:
removed from this PR adding in a different one.

Reply:
https://blocky.atlassian.net/browse/BKY-6437